templates/taxi/car_form.html

```diff
@@ -7,6 +7,15 @@ <h1>{{ object|yesno:"Update,Create" }} car</h1>
     {% csrf_token %}
     {{ form|crispy }}
 
-    <input type="submit" value="Submit" class="btn btn-primary">
+    <div>
+      <input type="submit" value="Submit" class="btn btn-primary">
+      {% if car %}
```

Comment:
Ensure that the `car` context variable is passed to this template when updating a car. This is essential for the conditional logic to function correctly, providing the correct URL for the 'Cancel' button.